chord_metadata_service/phenopackets/models.py

```diff
@@ -98,33 +103,41 @@ class PhenotypicFeature(models.Model, IndexableMixin):
     phenopacket = models.ForeignKey(
         "Phenopacket", on_delete=models.CASCADE, blank=True, null=True, related_name='phenotypic_features')
     extra_properties = JSONField(blank=True, null=True, help_text=rec_help(d.PHENOTYPIC_FEATURE, "extra_properties"))
-    created = models.DateTimeField(auto_now=True)
-    updated = models.DateTimeField(auto_now_add=True)
 
     def __str__(self):
         return str(self.id)
 
 
-class Procedure(models.Model):
-    """
-    Class to represent a clinical procedure performed on an individual
-    (subject) in order to extract a biosample
+# class Procedure(BaseTimeStamp, IndexableMixin):
+#     """
+#     Class to represent a clinical procedure performed on an individual
+#     (subject) in order to extract a biosample
 
-    FHIR: Procedure
-    """
+#     FHIR: Procedure
+#     """
+
+#     code = JSONField(validators=[ontology_validator], help_text=rec_help(d.PROCEDURE, "code"))
+#     body_site = JSONField(blank=True, null=True, validators=[ontology_validator],
+#                           help_text=rec_help(d.PROCEDURE, "body_site"))
+#     performed = JSONField(blank=True, null=True, validators=[
+#         JsonSchemaValidator(schema=TIME_ELEMENT_SCHEMA)])
+#     extra_properties = JSONField(blank=True, null=True, help_text=rec_help(d.PROCEDURE, "extra_properties"))
+
+#     def __str__(self):
+#         return str(self.id)
 
-    code = JSONField(validators=[ontology_validator], help_text=rec_help(d.PROCEDURE, "code"))
-    body_site = JSONField(blank=True, null=True, validators=[ontology_validator],
-                          help_text=rec_help(d.PROCEDURE, "body_site"))
-    extra_properties = JSONField(blank=True, null=True, help_text=rec_help(d.PROCEDURE, "extra_properties"))
-    created = models.DateTimeField(auto_now=True)
-    updated = models.DateTimeField(auto_now_add=True)
+
+class File(BaseTimeStamp, IndexableMixin):
+    uri = models.URLField(primary_key=True, max_length=200, help_text=rec_help(d.FILE, "uri"))
+    individual_to_file_identifiers = JSONField(blank=True, null=True,
+                                               help_text=rec_help(d.FILE, "individual_to_file_identifiers"))
+    file_attributes = JSONField(blank=True, null=True, help_text=rec_help(d.FILE, "file_attributes"))
 
     def __str__(self):
-        return str(self.id)
+        return str(self.uri)
```

Comment:
test

Reply:
I think we should still skip these, and perhaps later we can add an option to render them based on experiment metadata when we have URIs for those - but either way I don't think we need a model.